new-dti-website/components/initiatives/InitiativeDisplay.tsx

```diff
@@ -49,7 +49,7 @@ const Initiative = ({
           width >= LAPTOP_BREAKPOINT ? inter.className : ibm_plex_mono.className
         }`}
       >
-        {parts.map((part, index) => (part === eventName ? <b key={index}>{part}</b> : part))}
+        {parts.map((part, index) => part)}
```

Comment:
Wait can't we just do parts here then?

Reply:
LOL i accidentally edited smth oops

Reply:
fixed [here](https://github.com/cornell-dti/idol/pull/787/commits/b67a197c6c1002475ecf9cc76f3ea5ea402ccdf0)

Reply:
ok wait